_includes/header.html

```diff
@@ -37,6 +35,16 @@
         </li>
         <li><a href="{{ site.baseurl }}/blog">Blog</a></li>
       </ul>
+      <ul class="nav navbar-nav navbar-right">
+      <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project <span class="caret"></span></a>
+        <ul class="dropdown-menu">
+          <li><a href="{{ site.baseurl }}/presentation-materials/">Presentation Materials</a></li>
+          <li><a href="{{ site.baseurl }}/material/">Material</a></li>
```

Comment:
Can we think of a better name than Material or Project Material? Sounds very generic to me, and doesn't necessarily lead me what the actual content is. My preference would be Press material, or something similar. I'm fine with Logos, or other similar ideas.